whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+angles $\{0, 0, 0, 0, \pi, \pi, \pi, \pi\}$ is zero, but the distribution is far
+from uniform.  The Kolmogorov-Smirnov (KS) test is well known for investigating
+whether a set of data are consistent with a given distribution.  The KS
+statistic, off which the test is based, is defined as the maximum absolute
+difference in the empirical cumulative distribution function (CDF) of the data
+and the CDF of the distribution being tested.  The Kuiper statistic is a slight
+modification of the KS statistic, defined as the sum of the maximum difference
+and absolute minimum (maximally negative) difference between the empirical and
+test CDFs.  This modification is convenient for characterizing distributions
+defined on a circle, since it makes the statistic invariant under rotations of
+the data.  The larger the Kuiper test statistic (which ranges between 0 and 1),
+the larger the difference between the empirical distribution and the test
+distribution.  To incorporate the spin-2 nature of shear systematics in the
+Kuiper statistic, we map the values $\theta_i \rightarrow \theta_i \mod \pi$ and
+compare to the uniform distribution between 0 and $\pi$.
 
-We can use largescale patterns of distortions of the PSF over the 20,000 stars per exposure for
-PSF regularization in the per-CCD PSF fitting. In the per CCD fits, there is a benefit to
-setting aside some stars for validation tests of PSF extrapolation. 
-In addition to using all the stars in a given visit, there is useful information in the
-wavefront sensors and the guide CCDs that may be used to regularize the PSF
-reconstruction in a visit. We might read out guider CCDs in different ways to better
-monitor the atmosphere.
 
 \subsection{OpSim Analysis}
 
-\begin{figure}
-\centering\includegraphics[width=\linewidth]{figs/enigma1189RmsAnglerotSkyPosugrizybandallpropsOPSIComboHistogram.png}
-\caption{The relative angle of the detector plane with respect to the sky, RotSkyPos, as a histogram showing the number of fields vs. rms of the parameter.}
-\label{RotSkyPos}
+The distribution of AngularSpread for $2 \times$ rotSkyPos is shown in
+\autoref{fig:WL_AngularSpread_rotSkyPos} for the lastest baseline OpSim run,
+minion\_1016.  The left panel shows a sky map for the i-band (in this and the
+following figures, the sky maps vary only minimally between the two principal
+lensing filters, $r$ and $i$), while the right panel shows a histogram of values
+for each LSST filter.  The distribution of the Kuiper statistic for rotSkyPos
+mod $\pi$ is similarly shown in \autoref{fig:WL_Kuiper_rotSkyPos}. Using
+techniques similar to \citet{Jee&Tyson2011}, they did a study of the shear
+residual systematics due to known LSST CCD brighter-fatter anisotropy in 100
+revisits to a single field with random angular orientations and seeing sampled
+from the expected distribution. The needed factor of 10 suppression of the
+CCD-based shear systematic was obtained, reaching the SRD floor on cosmic shear
```

Comment:
Sorry if I'm being dense, but is this a statement that the suppression was achieved without any correction for b-f anisotropy?  Or is this a statement about suppression of the known residuals after b-f corrections (which are currently imperfect)?